### PR TITLE
4032 - Update Country Collaborators

### PR DESCRIPTION
**Pull request by sorja**
- Remove invitations table
- Merge columns/components from invitations to users table
  - Info, Copy link, Remove [invitation], Resend [invitation]
- Remove unused components (ActionsContainer)
- Update params and props to use type `CountryUserSummary` over `UserInvitation`

### CODE REVIEW

File: src/server/api/user/index.ts [hunk 1]
```diff
@@ -31,6 +31,10 @@ export const UserApi = {
     express.get(ApiEndPoint.User.invitation(), getInvitation)
     express.get(ApiEndPoint.User.invitationAccept(), acceptInvitation)
     express.get(ApiEndPoint.User.invitationSendEmail(), AuthMiddleware.requireInviteUser, sendInvitationEmail)
+    /**
+     * @deprecated
+     */
+    // TODO Delete
```
Comment (sorja) [inline src/server/api/user/index.ts:37]: Next PR, reminder to self

File: src/client/pages/CountryHome/Collaborators/UserList/Users/Actions/Actions.tsx [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
 }
 
 const Actions: React.FC<Props> = (props: Props) => {
-  const { user } = props
+  const { countryUserSummary } = props
   return (
-    <ActionsContainer>
-      <Edit user={user} />
-    </ActionsContainer>
+    <div className="actions-container">
```
Comment (sorja) [inline src/client/pages/CountryHome/Collaborators/UserList/Users/Actions/Actions.tsx:18]: Bring all action buttons from previous UserInvitation table (which was underneath) and update them to use countryUserSummary param

File: src/client/pages/CountryHome/Collaborators/UserList/Users/Users.tsx [hunk 3]
```diff
@@ -2,53 +2,74 @@ import './Users.scss'
...
       },
       {
-        component: ({ datum }) => <span>{datum.email}</span>,
+        component: ({ datum }) => {
+          const { role, invitation } = CountryUserSummaries.getCountryRoleAndInvitation(datum, countryIso)
+          const label = datum.email
+          const _classNames = { invitation: Boolean(!role && invitation) }
+
+          return <span className={classNames(_classNames)}>{label}</span>
+        },
```
Comment (sorja) [inline src/client/pages/CountryHome/Collaborators/UserList/Users/Users.tsx:62]: Left duplicated code on purpose, so we can refactor the style later and introduce `Field` or `Text` or similar component

**Review by minotogna — APPROVED** [review]
👍

**Pull request merged**